### DISCUSSION TITLE
2009/05/21/%E3%80%8A%E6%88%91%E5%92%8Clabview%E3%80%8B%E5%86%85%E5%AE%B9%E4%BB%8B%E7%BB%8D

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#264] [Announcements]
# 2009/05/21/%E3%80%8A%E6%88%91%E5%92%8Clabview%E3%80%8B%E5%86%85%E5%AE%B9%E4%BB%8B%E7%BB%8D

本书特点

https://blog.qizhen.xyz/2009/05/21/%E3%80%8A%E6%88%91%E5%92%8Clabview%E3%80%8B%E5%86%85%E5%AE%B9%E4%BB%8B%E7%BB%8D

**Comment by ruanqizhen** [comment]
之前博客下的留言：

Shaodong
2009年06月2日 11:01 编辑
期待啊期待，期待你把面向对象编程部分写得详细点~

回复
Zhen
2009年05月21日 21:59 编辑
期待早日上市：）

回复
